### PR TITLE
README update

### PR DESCRIPTION
**Pull request by Dr-N0**
Not done yet, just creating the pr

### CODE REVIEW

**Comment by Dr-N0** [comment]
Ignore the code practices section

File: README.md [hunk 1]
```diff
@@ -32,11 +33,40 @@ docker run --rm -i -t -p 5000:8080 --name=schedulemaker schedulemaker
...
+	    "graceful-fs": {
+	        "version": "4.2.2"
+	     }
+	  }
+	}
+	```
+	- Then run `npm install`
+- If your JS/CSS won't load
+	- Make sure you are on CSH's network. If not, try contacting a current maintainer.
+	- Check that your config file is filled out.
```
Comment (devinmatte) [inline README.md:57]: What is the reason for this?
Also CSH's network is required for the database, but if we give them a copy of the SQL that isn't a requirement

Comment (devinmatte) [reply README.md:57]: I've never had to set up the shrinkwrap. If you're running node 10.x it should work fine (set things up a few weeks ago from scratch on my work mac with no issues

Comment (Dr-N0) [reply README.md:57]: Could just be a product of my environment, but the graceful stuff had to be updated separately, and this was the method I found to do it

Comment (Dr-N0) [reply README.md:57]: @devinmatte

Comment (devinmatte) [reply README.md:57]: If you want this merged, remove the notes about shrinkwrap

Comment (Dr-N0) [reply README.md:57]: Making changes

**Comment by Dr-N0** [comment]
I don't have the original code in a branch, so without reverting I'm gonna review the changes and just re PR

**Pull request closed without merging**